tests/components/lcn/test_config_flow.py

```diff
@@ -0,0 +1,56 @@
+"""Tests for the LCN config flow."""
+from unittest.mock import patch
+
+from homeassistant import data_entry_flow
+from homeassistant.components.lcn import config_flow
+from homeassistant.config_entries import SOURCE_IMPORT
+from homeassistant.const import CONF_HOST, CONF_IP_ADDRESS, CONF_PORT
+
+from tests.common import MockConfigEntry
+
+
+def init_config_flow(hass):
+    """Init a configuration flow."""
+    flow = config_flow.LcnFlowHandler()
```

Comment:
Please use the core interfaces to start the test. Use our scaffold script to generate best practice code including config flow tests.

Reply:
I hope I have done everything as supposed, now.